easybuild/easyconfigs/h/h5py/h5py-2.5.0-CrayGNU-2016.11-Python-2.7.12-parallel18.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "PythonPackage"
+
+name = 'h5py'
+version = '2.5.0'
+python = 'Python'
+pyver = '2.7.12'
+pyshortver = '.'.join(pyver.split('.')[0:2])
+hdf5suff = '-parallel18'
+versionsuffix = '-%s-%s%s' % (python, pyver, hdf5suff)
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+dependencies = [
+    (python, pyver),
+    ('cray-hdf5-parallel/1.8.16', EXTERNAL_MODULE),
+    ('mpi4py', '1.3.1', '-Python-%(pyver)s' + '-timed-pingpong'),
+]
+
+prebuildopts = ' python setup.py configure --mpi --hdf5-version=1.8.16 --hdf5=/opt/cray/pe/hdf5-parallel/1.8.16/gnu/51/ && '
```

Comment:
It should.

If it is not set, it should be because this version is missing here: https://github.com/eth-cscs/production/blob/master/easybuild/cray_external_modules_metadata.cfg